common/src/main/java/com/wynntils/wynn/model/LootrunModel.java

```diff
@@ -600,7 +607,8 @@ public List<LootrunInstance> getLootruns() {
             }
         }
 
-        return lootruns;
+        LOOTRUN_INSTANCE_CACHE = lootruns;
```

Comment:
perhaps a check if the files have changed using md5s might be better for performance

Reply:
I didn't want to overengineer this. As this is done async, it is fast enough, and doesn't lag the GUI.

Reply:
ok